src/bp/lang-server/api.ts

```diff
@@ -124,22 +124,38 @@ export default async function(options: APIOptions) {
       domain: options.service.domain,
       readOnly: options.readOnly,
       languages: options.service
-        .listModels()
+        .listFastTextModels()
         .filter(x => x.loaded)
         .map(x => x.name)
     })
   })
 
   app.post('/vectorize', waitForServiceMw, async (req, res, next) => {
+    try {
+      const input = req.body.input
+      const language = req.body.lang || 'en'
+
+      if (!input || !_.isString(input)) {
+        throw new BadRequestError('Param `input` is mandatory (must be a string)')
+      }
+
+      const [vectors, tokens] = await options.service.vectorize(input, language)
```

Comment:
We can leave it as this for the moment as this is not a real issue.

Seems like you have one service function for each api route which is not "a good separation of concerns" if I can use the expression. I'd rather like if you could call the 2 functions independently, then returning you composed results. You don't have to do it for now but keep that in mind.